packages/terra-collapsible-menu-view/CHANGELOG.md

```diff
@@ -3,11 +3,14 @@ ChangeLog
 
 Unreleased
 ----------
+### Fixed
+* Fixed divider transition between normal and in-menu view.
```

Comment:
This log should be moved under the other "Fixed" section.

Reply:
Can we do that..? Please let me know if i am wrong, I think while release it will be moved under single header until then this should be in the order changes were made.